### PR TITLE
Adjust packet size to use number of bytes

### PR DESCRIPTION
**Pull request by claywar**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x-ish] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

Updates packet size to use number of bytes instead of byte pairs where packet's setSize function is newly used.

### CODE REVIEW

File: src/map/packets/macroequipset.cpp [hunk 1]
```diff
@@ -34,7 +34,7 @@ CAddtoEquipSet::CAddtoEquipSet(uint8* orig)
     // Should Push 0x116 (size 68) in responce
     // 0x04 is start, contains 16 4 byte parts repersently each slot in order
     this->setType(0x16); // TODO
```
Comment (zach2good) [inline src/map/packets/macroequipset.cpp:36]: Is this one of my TODOs to change this to 116?

Comment (claywar) [reply src/map/packets/macroequipset.cpp:36]: Yeah, the ones I left are prefixed with  `// TODO: Verify`

**Review by zach2good — APPROVED** [review]
I'm happy for this to go ahead, since it is targeting my other PR branch for this 👍 

We're going to need to spend time going through each packet anyway

**Comment by zach2good** [comment]
TODO: A little documentation on `setSize()` to say what the old style was, what the new style is, etc.

**Pull request merged**